runconfig/config.go

```diff
@@ -30,6 +31,7 @@ type Config struct {
 	Cmd             *stringutils.StrSlice // Command to run when starting the container
 	Image           string                // Name of the image as it was passed by the operator (eg. could be symbolic)
 	Volumes         map[string]struct{}   // List of volumes (mounts) used for the container
+	CLIVolumeSpecs  map[string]struct{}   // CLI-passed list of volumes (mounts) used for the container
```

Comment:
@cpuguy83 & I discussed this on IRC yesterday. I changed it to CLIVolumeSpecs. VolumeSpecs was Brian's suggestion. I added CLI as I wanted to make it clear that this field is only intended to be used by the docker CLI, not by a REST caller (and it's commented as such). I really don't care what it's called though.